docs/src/Typeahead.doc.js

```diff
@@ -106,6 +106,13 @@ card(
           'Forward the ref to the underlying component container element',
         href: 'refExample',
       },
+      {
+        name: 'zIndex',
+        type: 'interface Indexable { index(): number; }',
+        description:
+          'An object representing the zIndex value of the Typeahead list. Only use when Typeahead is overlayed by parent Layer components with zIndex.',
+        href: 'zIndex',
```

Comment:
I would update this as well. Maybe: 'An object representing the z-index value of the Typeahead list. Only use when Typeahead is used within a parent component that has a z-index set'

Reply:
DONE